src/modules/enrollment/components/dashboardPages/EnrollmentCurrentLivingSituationsPage.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box } from '@mui/system';
+
+import TitleCard from '@/components/elements/TitleCard';
+import { useEnrollmentDashboardContext } from '@/components/pages/EnrollmentDashboard';
+import NotFound from '@/components/pages/NotFound';
+
+const EnrollmentCurrentLivingSituationsPage = () => {
+  const { enrollment } = useEnrollmentDashboardContext();
+  const enrollmentId = enrollment?.id;
+  const clientId = enrollment?.client.id;
+
+  if (!enrollment || !enrollmentId || !clientId) return <NotFound />;
+
+  return (
+    <>
+      <TitleCard title='Current Living Situations' headerVariant='border'>
+        <Box p={3}>Not yet implemented</Box>
+      </TitleCard>
+    </>
```

Comment:
Extra encapsulating fragment
```suggestion
    <TitleCard title='Current Living Situations' headerVariant='border'>
      <Box p={3}>Not yet implemented</Box>
    </TitleCard>
```